### ISSUE TITLE
pcf apply-changes should dynamically determine errands to run

### ISSUE THREAD

**Issue by guidowb** [cf-platform-eng/tile-generator#72]
```
➜  pez-04 pcf apply-changes
- 500
{"error":"Unknown errand: deploy-all"}
➜  pez-04 
```

**Comment by guidowb** [comment]
Also, when including bosh releases with errands other than deploy-all and delete-all, these are not currently being run

**Comment by guidowb** [comment]
This is better supported by the 1.8 API and solved for that. Will likely close this issue without a fix for older releases once 1.8 is ubiquitous.

**Comment by cholick** [comment]
#94 fixes this, yeah?

**Comment by guidowb** [comment]
Fixed by #94.

**Comment by guidowb** [comment]
This works on 1.8, and we have no plan to address it on older releases. For pipelines against older versions, the workaround is to explicitly name the errands to run on `pcf apply-changes`.
